src/helpers/cauldron/position.js

```diff
@@ -0,0 +1,93 @@
+import { BigNumber } from "ethers";
```

Comment:
Can you add unit tests for cauldron creation?

Reply:
To the extent possible, we probably need some kind of end-to-end testing at a higher level (before you even merge in the changes) so we can easily compare before and after states.

Reply:
In the past, I've used the nock library to handle stubbing of network calls... Something you might want to consider, but it might be better to just stub with mocha or something.

https://www.npmjs.com/package/nock#recording

Reply:
Would be great to mimic the Uniswap v3 repo: https://github.com/Uniswap/interface

Jest test files are in the respective src folders (e.g., `interface/src/utils`) and then they have a separate Cypress folder with a set of e2e tests.